.npmrc

```diff
@@ -0,0 +1 @@
+# IMPORTANT: DO NOT DELETE THIS. IT IS REQUIRED TO PASS THE BUILD FOR SOX COMPLIANT PACKAGES IN THE BB PIPELINE
```

Comment:
Of course it is...
![image](https://user-images.githubusercontent.com/573668/179891391-4a72243b-7c22-4c53-a971-5e0d7b8175f7.png)

Reply:
lol... I know right. I can't believe this is what I needed to do to get this to work.

Reply:
Actually, if we need to have it, just copy/paste the contents of `.yarnrc` into here and delete that file.  No need for duplication.  Yarn should read from `.npmrc` if `.yarnrc` is not available :)

Reply:
Just tried doing this and it looks like the BB pipeline is going to fail. It's hanging on the deploy step. So not sure this is a viable solution (the implemented solution was suggested by 2 engineers from the build team during a zoom call yesterday).